vm-image-builder: update vm-image-builder (#2630)
kubevirt/project-infra: images/vm-image-builder/README.md

vm-image-builder image
===============

KubeVirt CI general purpose image for building Customized ephemeral container-disk images for KubeVirt VM's.

To be more specific, for building VM images in kubevirtci/cluster-provision/images/vm-image-builder.

How to setup the image
-----------------

> The container is required to run in privileged mode. The shell script `start_libvirt.sh` would start the libvirtd related daemon, then libvirt related cli can works.

Reviewer: Please comment on why we need to run privileged.

vm-image-builder image
===============

KubeVirt CI general purpose image for building Customized ephemeral container-disk images for KubeVirt VM's.

To be more specific, for building VM images in kubevirtci/cluster-provision/images/vm-image-builder.

How to setup the image
-----------------

> The container supposed to run in privileged mode. The shell script `start_libvirt.sh` would start libvirtd related daemon,

Reviewer 2: ```suggestion
The container is required to run in privileged mode. The shell script `start_libvirt.sh` starts the libvirtd related daemon,
```
